assets/sass/components/_links.scss

```diff
@@ -211,7 +211,7 @@ Style guide: Link styles.1 Hover links
   }
 }
 
-main > article {
+article {
```

Comment:
@petronbot Actually… should this just read `main`? This has been a pain point for numerous peeps (including myself during the DTA website rework).